src/game/camera.c

```diff
@@ -1135,7 +1137,7 @@ void mode_8_directions_camera(struct Camera *c) {
     }
 #ifdef PARALLEL_LAKITU_CAM
     // extra functionality
-    else if (gPlayer1Controller->buttonPressed & U_JPAD) {
+    else if ((gPlayer1Controller->buttonPressed & U_JPAD) && (gMarioState->action != ACT_DEBUG_FREE_MOVE)) {
```

Comment:
we should not really have camera-specific behavior for debug free move, just wasting code and perf considering that no final release will have it

Reply:
what interference? this code is only when dpad up is being pressed. just dont press dpad up? its so pointless

Reply:
D-Pad Up is used to move up in debug free mode (and enter the mode in the first place), that's pretty damn important.

Reply:
i forgor

Reply:
This check should also be applied to `D_JPAD` actually